### PR TITLE
chore(splunk): update link to documentation 

### PR DESCRIPTION
**Pull request by vkrizan**
Updates link to documentation regarding setting up 3rd party integrations (Splunk).
Refactors/removes unneeded conditionals for documentation URL.

EVNT-463

### CODE REVIEW

**Comment by vkrizan** [comment]
@josejulio can you please have a look? Thanks

File: src/properties/Messages.ts [hunk 1]
```diff
@@ -34,7 +34,8 @@ const MutableMessages = {
                 description: 'To finish the Splunk configuration, follow the instructions to start the automation process.',
                 help: 'Configure the integration between your Splunk instance' +
                       ' and Red Hat Insights to allow you to receive events from Insights.',
-                helpUrl: null
+                helpUrl: 'https://access.redhat.com/documentation/en-us/red_hat_hybrid_cloud_console/2022/html-single' +
+                         '/configuring_notifications_and_integrations_on_the_red_hat_hybrid_cloud_console '
```
Comment (josejulio) [inline src/properties/Messages.ts]: nitpick
```suggestion
                         '/configuring_notifications_and_integrations_on_the_red_hat_hybrid_cloud_console'
```

Comment (vkrizan) [reply src/properties/Messages.ts]: Done. Thanks!

**Review by josejulio — APPROVED** [review]
lgtm

**Pull request merged**